client/data/disputes/resolvers.js

```diff
@@ -3,7 +3,8 @@
 /**
  * External dependencies
  */
-import { apiFetch, dispatch } from '@wordpress/data-controls';
+import { dispatch } from '@wordpress/data';
```

Comment:
The `dispatch` function from `@wordpress/data-controls` has been deprecated in favor of the one from `@wordpress/data`.

Reply:
👍 Addresses https://github.com/Automattic/woocommerce-payments/issues/2343 (partially?)

Reply:
yes! 🙌